sourced/ml/utils/spark.py

```diff
@@ -64,10 +64,14 @@ def add_spark_args(my_parser, default_packages=None):
 def create_spark(session_name,
                  spark=SparkDefault.MASTER_ADDRESS,
                  spark_local_dir=SparkDefault.LOCAL_DIR,
-                 config=SparkDefault.CONFIG,
-                 packages=SparkDefault.PACKAGES,
+                 config=None,
```

Comment:
You should make `SparkDefault.CONFIG` and `SparkDefault.PACKAGES` tuples and revert the default values back. We never alter the defaults. If we do then this is an error.

Reply:
Ok, I agree. We alter them everywhere, so I just revert it here and make an another PR to fix it